app/src/main/java/de/tum/in/tumcampusapp/managers/EduroamManager.java

```diff
@@ -47,18 +47,21 @@ public EduroamManager(Context context) {
      *
      * @return true if eduroam is already setup, false otherwise
      */
-    public boolean isConfigured() {
-        WifiManager wifiManager = (WifiManager) mContext.getSystemService(Context.WIFI_SERVICE);
+    static public WifiConfiguration getEduroamConfig(Context c) {
+        WifiManager wifiManager = (WifiManager) c.getApplicationContext().getSystemService(Context.WIFI_SERVICE);
         List<WifiConfiguration> list = wifiManager.getConfiguredNetworks();
+
+        //We didn't get a list, so maybe theres no wifi?
         if (list == null) {
-            return true;
+            return null;
```

Comment:
A little bit overhead is probably worth avoiding future NullPointerExceptions…
I'll hold you responsible for any future NPEs :P